practice/src/index.html

```diff
@@ -142,5 +142,67 @@ <h4 class="secondary-title">Popular Trip Packages</h4>
       </div>
     </section>
   </main>
+
+  <footer class="footer-section">
+    <!-- Footer section -->
+    <div class="footer-wrapper"> 
```

Comment:
@JennyDoJD could you help double the layout? It seems some div tags are unnecessary